docs/UserGuide.md

```diff
@@ -112,6 +112,10 @@ ConcertPhonebook is a **desktop app** for **Concert Organisers** to manage your
 - If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 
 - INDEX parameters have a max positive value of `2147483647` any value larger would result in an invalid command format.
+
+- `NAME` of persons can only comprise alphanumeric characters and "/".<br> 
```

Comment:
This could be misunderstood as '/' is always allowed. Change this to mention that '/' is only allowed for s/o and d/o. Additionally, mention that s/o or d/o must be placed in the middle of alphanumeric characters (ie. name cannot start / end with s/o, d/o)